src/lib/calc-helpers.js

```diff
@@ -39,3 +39,13 @@ export const setSliderByIndex = (idx, val) => {
   const identifier = match[1];
   calculator.setExpression({ id, latex: `${identifier}=${val}` });
 };
+
+// gets current state of calculator instance
```

Comment:
Comments may not be necessary for these functions, they are perfectly described by their name

Reply:
Done